### PR TITLE
Form transfer agreement WIP

### PR DESCRIPTION
**Pull request by flisowna**
Contributes to https://trello.com/c/dxKIneyd/866-create-front-end-for-inventory-transfers-between-bases-orgs
### Included: 
* transfer agreement form
* transfers agreement table
* shipment form 
* shipments table
### Still missing:
* access to views/mutations adjusted to permissions
* tests
* qr scanner integration
* rethinking/improving UX flow
* update shipment
* receiving side UI of the shipment

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -6,23 +6,23 @@ repos:
...
-          - eslint-plugin-import@latest
-          - eslint-plugin-prettier@3.1.3
-        files: \.[jt]sx?$
-        types: [file]
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v2.2.1 # local node version must be higher than 10
-    hooks:
-      - id: prettier
-        types_or: [javascript, jsx, json, ts, tsx, css, scss]
+  # - repo: https://github.com/pre-commit/mirrors-eslint
```
[the remark this answers is not in the document]
Comment (spaudanjo) [reply .pre-commit-config.yaml:9]: There were some flaws, but I try to investigate/fix before we merge it

File: back/boxtribute_server/main.py [hunk 2]
```diff
@@ -16,3 +17,11 @@
     database=os.environ["MYSQL_DB"],
     unix_socket=os.getenv("MYSQL_SOCKET"),
 )
+
+from .db import db
+from .models.definitions.transfer_agreement import TransferAgreement
```
[the remark this answers is not in the document]
Comment (pylipp) [reply back/boxtribute_server/main.py]: We decided to let a dropapp migration handle the table creation, can you remove these additions?

**Comment by pylipp** [comment]
Since #286 (and previous work in dropapp) was merged, the required tables are set up in the staging database and the database dump is updated. So this PR can be rebased on master and finalized @flisowna @spaudanjo

**Comment by pylipp** [comment]
[2.0 Create Front-end for Inventory Transfers between bases / orgs](https://trello.com/c/dxKIneyd/866-20-create-front-end-for-inventory-transfers-between-bases-orgs)

**Comment by aerinsol** [comment]
@HaGuesto @vahidbazzaz is this still in progress, or has the work here been incorporated into other PRs? What should we do with this one?

**Comment by pylipp** [comment]
@aerinsol parts from this PR are currently being integrated in the work of @HaGuesto and @vahidbazzaz for implementing create-agreement and agreement-overview screens in a TDD way. Otherwise this PR will not be used anymore, especially it won't be merged ever.

**Comment by aerinsol** [comment]
@vahidbazzaz @HaGuesto please do reference this PR if needed and tag @spaudanjo and @flisowna in co-authored commits. Since this will never be merged I will close this PR.

**Pull request closed without merging**